tox.ini

```diff
@@ -66,3 +66,18 @@ deps={[testenv:optional]deps}
 envdir={[testenv:optional]envdir}
 recreate={[testenv:fast]recreate}
 setenv={[testenv:fast]setenv}
+
```

Comment:
Yeah, because with tox we get the environment with Django and all requirements installed already.